core/build.sbt

```diff
@@ -0,0 +1,16 @@
+tutSettings
+
+tutScalacOptions := Seq(
+  "-deprecation",
+  "-encoding", "UTF-8",
+  "-feature",
+  "-language:existentials",
+  "-language:higherKinds",
+  "-unchecked",
+  //"-Xfatal-warnings",  // Note: making unused imports an error breaks tut
```

Comment:
You should open an issue against tut. I was about to, but then I wasn’t sure where these settings take effect, so I couldn’t describe it appropriately.

Also, could we use `tutScalacOptions = compilerOptions -- Seq("-Xfatal-warnings", …)`? (and add as many missing options to `compilerOptions` as possible)

We should really write a SlamData sbt plugin that has all of our restrictive stuff (compiler options, warts, etc.) so we don’t have to duplicate that in each project.